mindsdb_native/libs/phases/data_cleaner/data_cleaner.py

```diff
@@ -41,7 +41,12 @@ def run(self):
 
         self._remove_missing_targets(df)
 
+        len_before_dedupe = len(df)
         if self.transaction.lmd.get('deduplicate_data'):
-            self._remove_duplicate_rows(df)
+           self._remove_duplicate_rows(df)
```

Comment:
must be 4 spaces (in context of this `if` statement) before `self._remove_duplicate_rows(df)`